### ISSUE TITLE
Button not working

### ISSUE THREAD

**Issue by amiraizat124** [dsafa/audio-band#222]
Shuffle and Repeat button not working with Spotify. Other buttons work fine.

**Comment by dsafa** [comment]
Does it work the other way? If you change the shuffle and repeat in Spotify, is it updated?

**Comment by amiraizat124** [comment]
Yes..if I shuffle or repeat in Spotify, the audioband update the icon.

**Comment by dsafa** [comment]
Thats interesting that it works the other way. I'll have to do some digging

**Comment by dsafa** [comment]
Ok so I did some more research and it appears that playback control functionality is only available to Spotify premium users which I wasn't aware of. Are you using a premium account?

The play/pause/skip buttons work because they use the desktop app directly.

**Comment by vos6434** [comment]
there's this discord mod called powercord, it has its own spotify controls and their shuffle and repeat buttons work fine without me needing premium.

**Comment by dsafa** [comment]
Interesting, I'll take a look at it. Hopefully I'm just mistaken about the restrictions.
